igor-web/src/main/java/com/netflix/spinnaker/igor/config/KeelConfig.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 Netflix, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.netflix.spinnaker.igor.config;
+
+import com.netflix.spinnaker.config.OkHttpClientConfiguration;
+import com.netflix.spinnaker.igor.IgorConfigurationProperties;
+import com.netflix.spinnaker.igor.keel.KeelService;
+import com.netflix.spinnaker.retrofit.Slf4jRetrofitLogger;
+import com.squareup.okhttp.OkHttpClient;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import retrofit.Endpoints;
+import retrofit.RestAdapter;
+import retrofit.client.OkClient;
+
+@ConditionalOnProperty("services.keel.base-url")
+@Configuration
+public class KeelConfig {
+
+  @Bean
+  RestAdapter.LogLevel retrofitLogLevel(
+      @Value("${retrofit.log-level:BASIC}") String retrofitLogLevel) {
+    return RestAdapter.LogLevel.valueOf(retrofitLogLevel);
+  }
+
+  @Bean
+  KeelService keelService(
+      OkHttpClientConfiguration okHttpClientConfig,
```

Comment:
It looks like `OkHttpClientConfiguration` is deprecated in favor of `OkHttp3ClientConfiguration` so new usages should probably use the new one.  (Though I'm not too familiar with why it was deprecated and what's involved in migrating.)

Reply:
It's part of the shift between Retrofit 1 and Retrofit 2. I had a PR open that split those things in Kork more clearly. I will re-open it and get it merged.

Reply:
@ezimanyi I'll admit, I couldn't find a good example of using `OkHttp3ClientConfiguration` so I was lazy. @robfletcher do you have a good example of how to do things the recommended way?

Reply:
Found one! Updated this.